book/forms.rst

```diff
@@ -224,9 +228,9 @@ controller::
         $task = new Task();
 
         $form = $this->createFormBuilder($task)
-            ->add('task', 'text')
-            ->add('dueDate', 'date')
-            ->add('save', 'submit', array('label' => 'Create Task'))
+            ->add('task', Type\TextType::class)
+            ->add('dueDate', Type\DateType::class)
+            ->add('save', Type\SubmitType::class, array('label' => 'Create Task'))
```

Comment:
here we use a line break for the array:
https://github.com/symfony/symfony-docs/pull/5800/files#diff-da27b730a6f0f3e3f23c45088b6113a0R98

in this line not